indexer/db/neon_tx_db.py

```diff
@@ -247,6 +261,9 @@ def from_tx(cls, neon_tx: NeonTxModel, neon_rcpt: NeonTxReceiptModel) -> Self:
             is_canceled=neon_rcpt.is_canceled,
             is_completed=neon_rcpt.is_completed,
             logs=cls._encode_event_list(neon_rcpt),
+            base_fee_per_gas=base_fee_per_gas,
+            chain_id=tx_chain_id,
+            priority_fee_per_gas=priority_fee_per_gas,
```

Comment:
_RecordWithBlock.to_meta() is used for loading a NeonTx from the db.
Please implement the logic